api/room_resource/schema.py

```diff
@@ -147,8 +147,17 @@ def resolve_resources(self, info):
         page = self.page
         per_page = self.per_page
         unique = self.unique
-        query = Resource.get_query(info)
-        active_resources = query.filter(ResourceModel.state == "active")
+        resource_labels = self.filter_data.get('resource_labels')
+        active_resources = Resource.get_query(info).filter(
+            ResourceModel.state == "active")
+        if resource_labels:
+            active_resources = active_resources.join(
+                RoomResourceModel).join(RoomModel)
+            for resourse_label in resource_labels.split(','):
+                active_resources = active_resources.filter(
```

Comment:
Good work @jesseinit, perhaps you could move the filtering logic to the `helpers/room_filter/room_filter.py` file. It contains other instances of filtering and might be better placed there.

Reply:
Thanks.